src/SFA.DAS.EmploymentCheck.Application/Services/EmploymentCheck/EmploymentCheckService.cs

```diff
@@ -100,5 +103,17 @@ public async Task<IList<EmploymentCheckCacheRequest>> CreateEmploymentCheckCache
 
             return employmentCheckRequests;
         }
+
+        public Data.Models.EmploymentCheck CreateEmploymentCheck(EmploymentCheckCacheRequest request)
+        {
+            var employmentCheck = new Data.Models.EmploymentCheck();
+
+            employmentCheck.Id = request.ApprenticeEmploymentCheckId;
+            employmentCheck.Employed = request.Employed;
+            employmentCheck.RequestCompletionStatus = request.RequestCompletionStatus;
+            employmentCheck.ErrorType = request.Employed == null ? "HmrcFailure" : null;
+
+            return employmentCheck;
+        }
```

Comment:
this doesn't belong here (service isn't a factory). move to a static constructor inside the `EmploymentCheck`

Reply:
Got an error using a static constructor in EmploymentCheck about it needing to be parameterless so used a static method instead.